src/runtime/loop.zig

```diff
@@ -41,6 +41,9 @@ pub const Loop = struct {
     // event are finished.
     events_nb: usize,
 
+    // Used to stop repeating timeouts when loop.run is called.
+    stopping: bool,
```

Comment:
@krichprollsch do operations on `events_nb` really need to be synchronized/atomic?  Isn't this all single threaded? 

If I'm wrong, I should add atomic operations around `stopping` too.

Reply:
Well I guess it was in case of future multi-threads use...
I'm not sure we should maintain it since many other things would break in this case...